pennylane/drawer/drawable_layers.py

```diff
@@ -104,18 +110,29 @@ def drawable_layers(ops, wire_map=None):
             is_mid_measure = True
             mapped_wire = wire_map[op.wires[0]]
             measured_wires[op.id] = mapped_wire
-            op_occupied_wires = {mapped_wire}
+
```

Comment:
We could probably pull out helper function:

```
op_occupied_wires = _get_occupied_wires(op, wire_map, cond_measurements)
```

Might help tighten up the code a little.